avetisyang.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+@author: avetisyang
+"""
+
+
+def avetisyang(df) -> str:
+    """
+    Parameters
+    ----------
+    Returns Data Frane with 4 dependent variables:
+	['MSSubClass','MSZoning', 'Street', 'Alley']
+
+    """
+    import pandas as pd
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import seaborn as sns
+
+
+    data = pd.read_csv("train.csv")

```

Comment:
The 'df' variable passed in to the function will contain the data, no need to read the file and pass it into 'data'.

Reply:
Most of the function depends on the "data" variable; it might be simpler to change the function's argument to "data" rather than search out all of the instances of "data" and change them to "df"